apps/dapp-console/.eslintrc.cjs

```diff
@@ -25,5 +25,6 @@ module.exports = {
       { varsIgnorePattern: '_', argsIgnorePattern: '_' },
     ],
     'react-refresh/only-export-components': ['off'],
+    'no-extra-semi': 'off'
```

Comment:
prettier adds a `;` to the beginning of immediately invoking functions which is the correct behavior to avoid ASI failure, and eslint has a silly rule `no-extra-semi` which is being brought in so the two are conflicting, turning this off for eslint

Reply:
should this be included in all our eslintrc.cjs files?